test/functional/test_site/testBoxes.md

```diff
@@ -0,0 +1,15 @@
+**Test: Boxes**
```

Comment:
Definitely see the purpose of the code block tests, what does this cover that `componentParserTest` misses though?

Reply:
Ah, does componentParserTest cover the rendering of markdown inside components?

I'll go check later.

Reply:
Ah yeah, I found that it covered. My bad 👍

Reply:
ok, the code blocks tests looks good 👍